client/src/components/HistoryViewer/tests/HistoryViewerVersion-test.js

```diff
@@ -1,76 +1,222 @@
+/* eslint-disable import/no-extraneous-dependencies */
 /* global jest, describe, it, expect */
 
 import React from 'react';
-import ReactTestUtils from 'react-addons-test-utils';
 import { Component as HistoryViewerVersion } from '../HistoryViewerVersion';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15.4/build/index';
 
-/**
- * Wrapper class to ensure the component is nested correctly before testing it
- */
-class TestHistoryViewerVersion extends React.PureComponent {
-  render() {
-    const NullComponent = () => <div />;
-
-    return (
-      <table>
-        <tbody>
-          <HistoryViewerVersion
-            StateComponent={NullComponent}
-            FormActionComponent={NullComponent}
-            {...this.props}
-          />
-        </tbody>
-      </table>
-    );
-  }
-}
+Enzyme.configure({ adapter: new Adapter() });
 
 describe('HistoryViewerVersion', () => {
-  let component = null;
+  const StateComponent = () => <div />;
+  const FormActionComponent = () => <div />;
+
+  let mockOnCompareMode;
+  let mockOnCompareSelect;
+  let mockOnSelect;
   let version = {};
 
-  describe('getAuthor()', () => {
-    beforeEach(() => {
-      version = {
-        Version: 3,
-        Published: false,
-        Author: {
-          FirstName: 'John',
-          Surname: 'Smith',
-        },
-        Publisher: {
-          FirstName: 'Sarah',
-          Surname: 'Smith',
-        },
-      };
-    });
+  beforeEach(() => {
+    mockOnCompareMode = jest.fn();
+    mockOnCompareSelect = jest.fn();
+    mockOnSelect = jest.fn();
 
-    it('returns the author name when unpublished', () => {
-      component = ReactTestUtils.renderIntoDocument(
-        <TestHistoryViewerVersion version={version} />
+    version = {
+      Author: {
+        FirstName: 'John',
+        Surname: 'Smith',
+      },
+      Published: false,
+      Publisher: {
+        FirstName: 'Sarah',
+        Surname: 'Smith',
+      },
+      Version: 3,
+    };
+  });
+
+  describe('HistoryViewerVersion', () => {
```

Comment:
We're already describing it at this point. This should be `describe('handleCompare()', ...`